### ISSUE TITLE
Ctx in conversation.external

### ISSUE THREAD

**Issue by idel28102001** [grammyjs/conversations#57]
Whenever I use ctx(such as reply) in conversation.external, my app "forgets" about all his comands.

There is message in telegram https://t.me/grammyjs/86807

**Comment by KnorpelSenf** [comment]
I still believe this to be a duplicate of #68. Feel free to reopen if the problem persists after #68 is fixed. Closing.
